src/wp-includes/default-constants.php

```diff
@@ -120,6 +120,15 @@ function wp_initial_constants() {
 		define( 'MEDIA_TRASH', false );
 	}
 
+	/**
+	 * Minimum size for a LCP candidate image. Size measusred in width x height.
+	 *
+	 * @since 6.3
+	 */
+	if ( ! defined( 'WP_LCP_MIN_IMAGE_SIZE' ) ) {
```

Comment:
@felixarntz @joemcgill I need name suggestion here. I wanted to avoid use of LCP in variable and function names, but i couldn't think of a better alternative.

Reply:
Regarding the naming, I can't think of a better name either at this point, but two separate important points:
* I think if we want to make this customizable, it should be a filter, not a constant.
* Potentially it's better for now to just hard-code the `50000` into the function, as that allows us to focus on the logic instead of having to think about what to name this. We can always add customization later, but it's harder to get rid of a bad name once we have introduced it :)

Reply:
I agree that a filter would be better here. Something like this could work:

```php
$wp_min_priority_img_pixels = apply_filters( 'wp_min_priority_img_pixels', 5000 );
```

I took a pass a name suggestion too, I think saying "priority image" is better, since this is specifically for determining priority hints. I also slightly prefer the word "pixels" to "size" since image size can be confused for file size instead of a number of intrinsic pixels, which is what this value is referring to.

Reply:
I have gone ahead and used the same name for filter.